news/2024/2024-02-16-osumania-7k-world-cup-2024-semifinals-recap.md

```diff
@@ -0,0 +1,106 @@
+
+<div class="osu-md__paragraph">
+    <audio controls>
+        <source src="https://assets.ppy.sh/artists/401/The%20Lost%20Symphonies/Akiri%20-%20Symphony%20Of%20The%20Lost%20Souls.mp3">
+    </audio>
+</div>
+
+[![](/wiki/shared/news/2024-02-16-osumania-7k-world-cup-2024-semifinals-recap/Voyage.jpg)](https://osu.ppy.sh/beatmapsets/2130624#mania/4480927)
+
+<div class="osu-md__paragraph">
+    <audio controls>
+        <source src="https://assets.ppy.sh/artists/392/Songs/Sydosys_-_Voyage.mp3">
+    </audio>
+</div>
+
+## Semifinals results
+
+As the final rounds draw nigh, all the teams are starting to show their true colours as we reach endgame levels of difficulty. As such, those who were able to keep up with the sharp increase in difficulty had an easier time this round. Because of that, some teams that initially did quite well in earlier rounds—despite giving their all — weren't able to make a dent on some of the behemoths they were up against.
+
+| Team A |  |  | Team B |
+| --: | :-: | :-: | :-- |
+| **Australia** ::{ flag=AU }:: | **6** | 1 | ::{ flag=US }:: United States |
+| **Japan** ::{ flag=JP }:: | **6** | 1 | ::{ flag=RU }:: Russian Federation |
+| **Thailand** ::{ flag=TH }:: | **6** | 0 | ::{ flag=PL }:: Poland |
+| China ::{ flag=CN }:: | 4 | **6** | ::{ flag=MY }:: **Malaysia** |
+| **Canada** ::{ flag=CA }:: | **6** | 1 | ::{ flag=BR }:: Brazil |
+| **Japan** ::{ flag=JP }:: | **6** | 4 | ::{ flag=AU }:: Australia |
+| **South Korea** ::{ flag=KR }:: | **6** | 0 | ::{ flag=PH }:: Philippines |
+| **Thailand** ::{ flag=TH }:: | **6** | 2 | ::{ flag=CA }:: Canada |
+
+## Match highlights
+
+### China ::{ flag=CN }:: vs. ::{ flag=MY }:: Malaysia
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=v2058868470&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
+</div>
+
+China vs. Malaysia was a long-awaited clash between two 7K titans. These two teams have been consistent contenders for the MWC7K podium. China, the 2016 champions — came into this match having narrowly escaped with a tiebreaker win in the Quarterfinals. The Malaysian duo, on the other hand, will finally be tested against a team of their calibre.
+
+These two countries are known as the 'noodle countries' due to their VSRG history. As such, this match was initially seen as a 'who will choke on their rice first' affair. As expected, Malaysia opened with LN3, [GHOST](https://osu.ppy.sh/beatmapsets/2130609#mania/4480895), but it was China who drew first blood. The noodle-slurping action continued as China started with LN2, [Vulnerability](https://osu.ppy.sh/beatmapsets/2130611#mania/4480897), which happened to be their protected pick for this match. However, Malaysia were able to snatch this table away from them. Both teams then traded break points on the next two charts to level the match at 2–2.
```

Comment:
```suggestion
These two are known as the "noodle countries" due to their VSRG history. As such, this match was initially seen as a "who will choke on their rice first" affair. As expected, Malaysia opened with LN3, [Hoshimachi Suisei - GHOST](https://osu.ppy.sh/beatmapsets/2130609#mania/4480895), but it was China who drew first blood. The noodle-slurping action continued as China started with LN2, [Vulnerability](Mili - https://osu.ppy.sh/beatmapsets/2130611#mania/4480897), which happened to be their protected pick for this match. However, Malaysia was able to snatch this table away from them. Both teams then traded break points on the next two charts to level the match at 2–2.
```

Reply:
```suggestion
These two countries are known as the 'noodle countries' due to their VSRG history. As such, this match was initially seen as a 'who will choke on their rice first' affair. As expected, Malaysia opened with LN3, [GHOST](https://osu.ppy.sh/beatmapsets/2130609#mania/4480895), but it was China who drew first blood. The noodle-slurping action continued as China started with LN2, [Mili - Vulnerability](https://osu.ppy.sh/beatmapsets/2130611#mania/4480897), which happened to be their protected pick for this match. However, Malaysia were able to snatch this table away from them. Both teams then traded break points on the next two charts to level the match at 2–2.
```
i think you put the text in the wrong place